### DISCUSSION TITLE
SipSorcery and .NET MAUI (iOS & Android)

### DISCUSSION THREAD

**Discussion by headintheclouds21** [sipsorcery-org/sipsorcery#1077] [General]
Hi devs,

I have a .NET MAUI app that's targeting mobile frameworks (iOS and Android) and I'd like to implement SipSorcery into my project. When digging through docs and different discussions i can see that it is compatible with desktop frameworks (linux, macos, windows); however, I'm not seeing much, if anything, about mobile development. Does anyone know if SipSorcery would be compatible with iOS and Android? Any advice would be greatly appreciated

**Comment by sipsorcery** [comment]
I've never gotten around to checkong out MAUI but there was a [successful prototype](https://github.com/sipsorcery-org/sipsorcery/tree/master/examples/SIPExamples/iOSSIPGetStarted) using Xamarin on iOS 4 years ago.
